sermant-plugins/sermant-tag-transmission/tag-transmission-servicecomb-plugin/src/test/java/com/huaweicloud/sermant/tag/transmission/servicecomb/interceptors/ServiceCombRpcConsumerInterceptorTest.java

```diff
@@ -46,38 +46,38 @@ public void doBefore(TrafficTag trafficTag) {
 
     @Test
     public void testServiceCombRpcConsumer() {
-        // 定义参数
+        // defineParameter
         ExecuteContext context;
         ExecuteContext returnContext;
         Map<String, String> expectContext;
 
-        // Invocation为null
+        // Invocation is null
         context = buildContext(null);
         returnContext = interceptor.before(context);
         Assert.assertNull(returnContext.getArguments()[0]);
 
-        // 流量标签透传开关关闭
+        // The traffic label pass-through switch is turned off
         tagTransmissionConfig.setEnabled(false);
         context = buildContext(new Invocation());
         returnContext = interceptor.before(context);
         expectContext = buildExpectContext();
         Assert.assertEquals(((Invocation) returnContext.getArguments()[0]).getContext(), expectContext);
         tagTransmissionConfig.setEnabled(true);
 
-        // TrafficTag包含完整的tag信息
+        // the traffic tag contains complete tag information
         context = buildContext(new Invocation());
         returnContext = interceptor.before(context);
         expectContext = buildExpectContext("id", "name");
         Assert.assertEquals(((Invocation) returnContext.getArguments()[0]).getContext(), expectContext);
 
-        // TrafficTag只有部分tag信息
+        // traffic tag indicates only partial tag information
         context = buildContext(new Invocation());
         TrafficUtils.getTrafficTag().getTag().remove("id");
         returnContext = interceptor.before(context);
         expectContext = buildExpectContext("name");
         Assert.assertEquals(((Invocation) returnContext.getArguments()[0]).getContext(), expectContext);
 
-        // TrafficTa没有tag信息
+        // traffic tag there is no tag information
```

Comment:
there is no tag information in TrafficTag

Reply:
done